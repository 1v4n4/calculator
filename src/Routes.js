import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';
import Calculator from './components/Calculator';
import Footer from './components/Footer';
import Header from './components/Header';
import Home from './components/Home';
import Quote from './components/Quote';

const Routes = () => (

  <BrowserRouter>
    <Header />
    <Switch>
      <Route exact path="/" component={Home} />
      <Route path="/Calculator" component={Calculator} />
      <Route path="/Quote" component={Quote} />
    </Switch>
    <Footer />
  </BrowserRouter>
);

export default Routes;
