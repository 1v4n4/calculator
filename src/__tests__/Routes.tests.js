import renderer from 'react-test-renderer';
import Routes from '../Routes';

it('renders website correctly', () => {
  const route = renderer.create(<Routes />).toJSON();
  expect(route).toMatchSnapshot();
});
