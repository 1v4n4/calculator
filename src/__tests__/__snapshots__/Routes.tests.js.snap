// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders website correctly 1`] = `
Array [
  <header>
    <div
      className="headlineDiv"
    >
      <h1>
        Math Magicians
      </h1>
    </div>
    <nav>
      <a
        className="navLink"
        href="/"
      >
        Home
      </a>
      <p
        className="navLink"
      >
        |
      </p>
      <a
        className="navLink"
        href="/Calculator"
      >
        Calculator
      </a>
      <p
        className="navLink"
      >
        |
      </p>
      <a
        className="navLink"
        href="/Quote"
      >
        Quote
      </a>
    </nav>
  </header>,
  <div
    className="homeDiv"
  >
    <h2>
      Welcome to our page!
    </h2>
    <p
      className="paragraph"
    >
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
    </p>
    <p
      className="paragraph"
    >
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
    </p>
  </div>,
  <footer>
    <p
      className="foot"
    >
      Made by
      <a
        href="https://github.com/1v4n4"
        id="gitLink"
        rel="noreferrer"
        target="_blank"
      >
         1v4n4 
      </a>
      Â© 2021
    </p>
  </footer>,
]
`;
